manager/integration/tests/test_node.py

```diff
@@ -1507,14 +1507,60 @@ def test_node_config_annotation_invalid(client, core_api,  # NOQA
        "Create Default Disk on Labeled Nodes" in settings.
     3. The node tag or disks won't be updated.
     """
-    setting = client.by_id_setting(SETTING_CREATE_DEFAULT_DISK_LABELED_NODES)
-    client.update(setting, value="true")
 
     nodes = client.list_node().data
     node_name = nodes[0].id
+    node = client.by_id_node(node_name)
 
     # Case1: The invalid disk annotation shouldn't
     # intervene the node controller.
+
+    # Case1.1: Multiple paths on the same filesystem is invalid disk
+    # annotation, Longhorn shouldn't apply this.
+
+    # make a clean condition for test to start.
+    cleanup_node_disks(client, node_name)
+
+    # patch label and annotations to the node.
+    core_api.patch_node(node_name, {
+        "metadata": {
+            "labels": {
+                CREATE_DEFAULT_DISK_LABEL:
+                    CREATE_DEFAULT_DISK_LABEL_VALUE_CONFIG
+            },
+            "annotations": {
+                DEFAULT_DISK_CONFIG_ANNOTATION:
+                    '[{"path":"/root","allowScheduling":false,' +
+                    '"storageReserved":1024,"name":"root-name"},' +
+                    '{"path":"' + DEFAULT_DISK_PATH +
```

Comment:
Don't need to use `DEFAULT_DISK_PATH`. We can just do `/root` and `/home`. Both are likely on the root fs.